src/transformer/sts-importer.ts

```diff
@@ -1,62 +0,0 @@
-/*!
```

Comment:
Probably we should update PR name so it's clear when we merge it why this was removed. Seems like this is not related the dependencies update.

Reply:
It is indeed not directly related, and I should have mentioned that to the group. I'll update the description.